conf/solr/9.3.0/schema.xml

```diff
@@ -232,6 +232,7 @@
     <field name="datasetValid" type="boolean" stored="true" indexed="true" multiValued="false"/>
```

Comment:
Can we please have a release note snippet?

It should say, roughly:

- There is a new search field called fileCount that can be searched to discover the number of files per dataset
- When upgrading, you need to update your Solr schema.xml file and reindex.

Reply:
done!

Reply:
Thanks!